pkg/postgres/walker/walker.go

```diff
@@ -212,6 +212,10 @@ func getPostgresOptions(tag string, topLevel bool, ignorePK, ignoreUnique, ignor
 			opts.Reference.Directional = true
 		case field == "ignore-fks":
 			opts.IgnoreChildFKs = true
+		case strings.HasPrefix(field, "type"):
+			typeName := field[strings.Index(field, "(")+1 : strings.Index(field, ")")]
```

Comment:
Maybe in a follow up, but would be cool to add this to the stringutils package as a GetBetween function